app/Http/Controllers/TestController.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Auth;
+use App\Models\Word;
+use App\Models\Test;
+use Carbon\Carbon;
+
+class TestController extends Controller
+{
+    public function index()
+    {
+        $user = Auth::user();
+        $tests = $user->tests->toArray(); 
+
+        return view('history', compact(['tests']));
```

Comment:
```suggestion
        return view('history', compact('tests'));
```
nếu chỉ truyền 1 tham số thì nên tôi suggest như này nhé